### ISSUE TITLE
On conflict key is not working

### ISSUE THREAD

**Issue by studentX** [larryxiao/peloton#73]
INSERT INTO test (key, value, flag, size) VALUES ('key', 'nitin', 11, 1) ON CONFLICT (key) DO UPDATE SET value = excluded.value, flag = excluded.flag, size = excluded.size
> Succeeds
INSERT INTO test (key, value, flag, size) VALUES ('key', 'nitin1', 1, 1) ON CONFLICT (key) DO UPDATE SET value = excluded.value, flag = excluded.flag, size = excluded.size
> Fails

**Comment by larryxiao** [comment]
Yes. I reported this issue https://github.com/cmu-db/peloton/issues/131.
I think it worked in previous versions though. I will try to debug this.
